content/recipes/interoperability/registering-swisslipids-identifiers.md

```diff
@@ -0,0 +1,162 @@
+# Registering SwissLipids identifiers in Wikidata
+
+ ````{panels_fairplus}
+:identifier_text: FCBxxx
+:identifier_link: 'https://w3id.org/faircookbook/FCBxxx'
+:difficulty_level: 5
+:recipe_type: hands_on
+:reading_time_minutes: 15
+:intended_audience: bioinformatician, data_scientist, data_engineer
+:has_executable_code: nope
+:recipe_name: Registering SwissLipids identifiers in Wikidata
+```` 
+
+## Main Objectives
+
+The objective of this recipe is to describe how external identifiers can be added to Wikidata in an automated.
+
+___
+
+
+## Requirements
+
+To record identifiers of some resource to make the content more findable and more interoperable, first a [Wikidata](https://www.wikidata.org/) property
+needs to be requested. That process is described [here](https://www.wikidata.org/wiki/Wikidata:Property_proposal).
+
+---
+
+
+## Main Content
+
+Making datasets more findable and databases more interoperable, it is recommended to share not just top-level metadata, but also record-level
+metadata. For example, for chemical databases, you can share your chemical compound identifiers as open data, while keeping the data itself
+in your own database. That allows other databases, like Wikidata, to link to your database. First, that makes your data more findable,
+but it also makes your data easier to integrate and therefore more interoperable.
+
+This recipe follows this idea and starts with a CCZero-licensed mapping file, linking InChIKeys and database identifiers.
+Recipe *InChI and SMILES identifiers for chemical structures* ([fcb:FCB007](https://w3id.org/faircookbook/FCB007)) explains how InChIKeys
+can be generated for chemical compounds in your database.
+
+As an example, this recipe demonstrates the approach of listing [SwissLipid](http://www.swisslipids.org/#/) {footcite}`Aimo2015SwissLipids`
+identifiers in Wikidata, developed at the 2021 BioHackathon Europe. Earlier, a [Swiss Lipid property for Wikidata](https://www.wikidata.org/wiki/Wikidata:Property_proposal/SwissLipids_identifier)
+was already proposed, later approved, and created in Wikidata [just before the 2020 BioHackathon Europe](https://www.wikidata.org/w/index.php?title=Property:P8691&oldid=1287579005).
+An existing Wikidata property is a requirement to adding the external identifiers.
```

Comment:
````suggestion

```{warning}
An existing Wikidata property is a requirement to adding the external identifiers.
```

```